rotkehlchen/chain/ethereum/modules/airdrops/decoder.py

```diff
@@ -73,6 +83,7 @@ def __init__(
             evm_inquirer=ethereum_inquirer,
             base_tools=base_tools,
             msg_aggregator=msg_aggregator,
+            vcow=A_VCOW.resolve_to_evm_token(),
```

Comment:
pass it as an asset normally and do the `resolve_to_evm_token()` in the constructor.